README.md

```diff
@@ -2,3 +2,136 @@
 
 This document explains our approach to working with Rubocop in dxw. We should
 in general:
+
+### Basics
+- use the latest version of rubocop
+- configure CI to run rubocop and fail if there are *any* violations
+- configure your editor to quickly and loudly complain about rubocop
+  violations
+  - otherwise it'll fail on CI and you'll feel silly
+  - or it'll fail too late and you'll end up re-writing code to make it pass
+- use the `.rubocop.yml` file in this repo as a starting point.
+- keep the `.rubocop.yml` file in this repo up-to-date with the latest version
+  of rubocop
+
+### Exceptions
+- avoid adding additional exceptions to the defaults unless there's a really
+  good reason.
+  - Even if we don't personally like some of the decisions there's value in
+    keeping the number of exceptions to the defaults to a minimum.
+- consider if any rules we do add should also be added to the default config
+  in this repo
+  - Differences between projects should be kept to a minimum.
+
+### Todos
+- avoid creating a `.rubocop_todo.yml` file for new projects
+  - we should be able to stick to the rules easily enough when building
+    starting from scratch
+- avoid manually editing the `.rubocop_todo.yml`
+  - this should be considered an autogenerated file which could be regenerated
+    at any point
+
+## Handling exceptions to rules
+
+### Scenario 1: a temporary exclusion
+
+Sometimes we'll need to write some temporary code, or create a complicated bit
+of code which we expect to be made simpler by future development. In this case
+it's acceptable to generate a `.rubocop_todo.yml` file (see below).
+
+A common example is when controller actions become more complicated and start
+exceeding the ABC metric limit, but it's not yet practical to extract business
+logic or presentation into objects.
+
+`.rubocop_todo.yml` should be avoided for more permanent exceptions because it
+generally excludes the whole file from a particular rule, when actually all we
+want is to exclude a single line or single function.
+
+### Scenario 2: a genuine exception
+
+There will inevitably be cases where it is not possible to stick to a rule
+without materially making the code more difficult to understand. Examples
+include:
+
+- Long regular expressions (violating `Metrics/LineLength`)
+- Long case statements (violating `Metrics/MethodLength`
+
+In most cases these should be excluded inline - e.g.
+
+``` ruby
+# rubocop:disable Metrics/LineLength
+  postcode =~ /^\s*((GIR\s*0AA)|((([A-PR-UWYZ][0-9]{1,2})|(([A-PR-UWYZ][A-HK-Y][0-9]{1,2})|(([A-PR-UWYZ][0-9][A-HJKSTUW])|([A-PR-UWYZ][A-HK-Y][0-9][ABEHMNPRVWXY]))))\s*[0-9][ABD-HJLNP-UW-Z]{2}))\s*$/i
+# rubocop:enable Metrics/LineLength
+```
+
+This has the downside of adding noise to into the application code, but the
```

Comment:
I think there's an extra "to" here?

Reply:
More to the point, I think this is a fair trade off

Reply:
Fixed inline